app/repositories/ApprovalRepository.scala

```diff
@@ -56,31 +55,6 @@ class ApprovalRepositoryImpl @Inject() (implicit mongoComponent: ReactiveMongoCo
     )
     with ApprovalRepository {
 
-  private def processCodeIndexName = "approval-secondary-Index-process-code"
-
-  override def ensureIndexes(implicit ec: ExecutionContext): Future[Seq[Boolean]] =
-    // If current configuration includes an update to the unique attribute of the processCode index, drop the current index to allow its re-creation
-    collection.indexesManager.list().flatMap { indexes =>
-      indexes
-        .filter(idx =>
-          idx.name == Some(processCodeIndexName) && !idx.unique
-        )
-        .map { i =>
-          logger.warn(s"Dropping $processCodeIndexName ready for re-creation, due to configured unique change")
-          collection.indexesManager.drop(processCodeIndexName).map(ret => logger.info(s"Drop of $processCodeIndexName index returned $ret"))
-        }
-
-      super.ensureIndexes
-    }
-
-  override def indexes: Seq[Index] = Seq(
```

Comment:
Need to keep the indexes() fn , its called by simple-reactivemongo to create the index if required